api/models.py

```diff
@@ -1,16 +1,43 @@
 # api/models.py
 from django.db import models
-from django.contrib.auth.models import User
+#from django.contrib.auth.models import User
 from django.utils import timezone
 
 
+class User(models.Model):
+    username = models.CharField(max_length=255)
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+    email = models.CharField(max_length=255)
+    phone_number = models.CharField(max_length=15)
+    password = models.CharField(max_length=255)
```

Comment:
Why are we saving passwords as is? I cannot see any hashing algorithms in serializers or views. Also I don't see the endpoints responsible for regular user authentication. Was authentication a part of your task?